src/wallet/wallet2.cpp

```diff
@@ -105,7 +105,7 @@ namespace
 struct options {
   const command_line::arg_descriptor<std::string> daemon_address = {"daemon-address", tools::wallet2::tr("Use daemon instance at <host>:<port>"), ""};
   const command_line::arg_descriptor<std::string> daemon_host = {"daemon-host", tools::wallet2::tr("Use daemon instance at host <arg> instead of localhost"), ""};
-  const command_line::arg_descriptor<std::string> password = {"password", tools::wallet2::tr("Wallet password"), "", true};
+  const command_line::arg_descriptor<std::string> password = {"password", tools::wallet2::tr("Wallet password (escape/quote as needed)"), "", true};
```

Comment:
The users who need this message are not the sort of users who will understand what the word 'escape' means in this sense. 

Maybe something more direct like "Put the password in quotes" (although to be fair I can't quite understand when a user might need to do this so as to clarify the message further)